package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "todo_list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
```

Comment:
- [ ] Please include webpack-dev-server as a development dependency, otherwise it will not be possible to run `npm start`. You can do that by simply running `npm i webpack-dev-server --save-dev`